tmpl/blog/mirage-seal.md

```diff
@@ -26,47 +35,53 @@ Although there are many entities that can sign a certificate with different proc
+If you're buying a certificate for a domain you have registered through Gandi (via the registered account), the rest of the process is pretty automatic.  You should shortly receive an e-mail with a subject like "Procedure for the validation of your Standard SSL certificate", which explains the process in more detail, but really all you need to do is wait a while (about 30 minutes, for me).  After the certificate has been generated, Gandi will notify you by e-mail, and you can download your certificate from the SSL management screen.  Click the magnifying glass next to the name of the domain for which you generated the cert to do so.
+
+Once you've downloaded your certificate, you may also wish to append the [intermediate certificates](https://en.wikipedia.org/wiki/Intermediate_certificate_authorities).  Here's a help page on [gathering intermediate certificates](https://wiki.gandi.net/en/ssl/intermediate).  Equipped with the intermediate certificates, append them to the signed certificate downloaded for your site to provide a full certificate chain.
 
-## Packaging Up a Site with Mirage-Seal
+## Packaging Up an HTTPS Site with Mirage-Seal
 
-Equipped with a private key and a certificate, let's make an HTTPS unikernel!  First, use `opam` to install `mirage-seal`.  (If `opam` or other MirageOS tooling aren't set up yet, check out the [instructions for getting started](http://openmirage.org/wiki/install).)
+Equipped with a private key and a certificate, let's make an HTTPS unikernel!  First, use `opam` to install `mirage-seal`.  If `opam` or other MirageOS tooling aren't set up yet, check out the [instructions for getting started](http://openmirage.org/wiki/install).
 
 ```
 opam install mirage-seal
 ```
 
-`mirage-seal` has a few required arguments.  (For more thorough documentation including network configuration settings, see `mirage-seal --help` or [mirage-seal's README file](https://github.com/mirage/mirage-seal/blob/master/README.md).)
+`mirage-seal` has a few required arguments. 
 
 * `--data`: one directory containing all the content that should be served by the unikernel.  Candidates for such a directory are the top-level output directory of a static site generator (such as `public` for octopress), the `DocumentRoot` of an Apache configuration, or the `root` of an nginx configuration.
 * `--keys`: one directory containing the certificate (`server.pem`) and key (`server.key`) for the site.
```

Comment:
Basically it would be nice if both the signed and self-signed sections end up with the reader having the same named piece of informations that can be used refered to in this section.